virtualenv/builders/base.py

```diff
@@ -44,7 +50,35 @@ def __init__(self, python, flavor, system_site_packages=False,
     def check_available(cls, python):
         raise NotImplementedError
 
+    def _get_base_python_bin(self):
```

Comment:
Well, I'm not reviewing that PR... ;-) I was assuming that the expectation was that *this* PR would be merged, not that the comments would be handled in the "big" PR. Is that not how you see it?

But regardless, glad that you had it covered. Ta.